osbuild/inputs.py

```diff
@@ -106,11 +108,12 @@ def map(self, ip: Input, store: ObjectStore) -> Tuple[str, Dict]:
 
 
 @contextlib.contextmanager
-def make_args_file(tmp, args):
-    with tempfile.TemporaryFile("w+", dir=tmp, encoding="utf-8") as f:
-        json.dump(args, f)
-        f.seek(0)
-        yield f.fileno()
+def make_args_and_reply_files(tmp, args):
+    with (tempfile.TemporaryFile("w+", dir=tmp, encoding="utf-8") as f_args,
+          tempfile.TemporaryFile("w+", dir=tmp, encoding="utf-8") as f_reply):
```

Comment:
It looks like [parenthesised context managers](https://bugs.python.org/issue12782) weren't supported until recently (though considered a bug).

Reply:
Thanks, I fixed it to use `\ ` now.